### PR TITLE
Judge applications

### PR DESCRIPTION
**Pull request by DianaPerkinsDesign**
Adds judge application form, which takes markdown and sends it to a special judge repo on GitHub for our reviewers to assess.

### CODE REVIEW

File: src/pages/judge-application.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+      render={(data) => {
+        const wardens = data.allHandlesJson.edges.map(({ node }) => {
+          return { value: node.handle, image: node.image };
+        });
+        fields[0].options = wardens;
+
+        return (
+          <DefaultLayout
+            bodyClass="judge-application"
+            pageTitle="Judge Application | Code 423n4"
```
Comment (adamavenir) [inline src/pages/judge-application.js:192]: Code4rena

File: src/styles/forms.css [hunk 2]
```diff
@@ -31,3 +31,102 @@ label {
...
+.thank-you ol,
+.thank-you p {
+  line-height: 150%;
+}
+
+.thank-you ol li {
+  margin-bottom: 20px;
+}
+
+/* TODO: finish this battle with flexbox */
```
Comment (adamavenir) [inline src/styles/forms.css:71]: :muscle: you got this

**Review by adamavenir — APPROVED** [review]
Great work. The lambda function looks more secure than the first one I wrote which @gpersoon hacked within minutes ;)

Approved with a non-blocking suggested change.

**Comment by CloudEllie** [comment]
Couple of minor text suggestions for the "high-severity" fields: I think we need to clarify that we're asking for findings *they* submitted, and I also think we could clarify the labels & descriptions a bit. 

How does this read to you? 

> ###First high severity finding link
> Link to a valid, high-severity finding you submitted to a Code4rena contest
> 
> ###First high severity finding details
> Describe how the above finding demonstrates your depth of knowledge

If you're good with these changes then they'd apply to the second & third pairs of fields as well.

**Comment by CloudEllie** [comment]
Oh! And can we change the text on the button so instead of "Apply for judge" it reads, "Apply to be a judge"?

**Pull request merged**